manager/integration/tests/test_ha.py

```diff
@@ -2779,14 +2780,11 @@ def test_autosalvage_with_data_locality_enabled(client, core_api, make_deploymen
     target_pod = \
         core_api.read_namespaced_pod(name=pod_names[0], namespace='default')
     wait_delete_pod(core_api, target_pod.metadata.uid)
-    pod_names = common.get_deployment_pod_names(core_api, deployment)
-    wait_pod(pod_names[0])
+    pod = common.wait_and_get_any_deployment_pod(core_api, deployment_name)
 
-    command = 'cat /data/test'
-    verified_data = exec_command_in_pod(
-        core_api, command, pod_names[0], 'default')
+    _md5sum = get_pod_data_md5sum(core_api, pod.metadata.name, data_path)
 
-    assert test_data == verified_data
+    assert md5sum == _md5sum
```

Comment:
Suggest using meaningful variables like `expected_test_data_checksum` and `test_data_checksum`.